openedx/adg/lms/applications/models.py

```diff
@@ -30,12 +36,43 @@ def __str__(self):
         return 'User {user_id}, application status id={id}'.format(user_id=self.user.id, id=self.id)
 
 
+class BusinessLine(TimeStampedModel):
+    """
+    Model to save the business lines offered by ADG.
+    """
+    title = models.CharField(max_length=255, )
+    logo = models.ImageField(
+        upload_to='business-lines/uploaded-logo/', null=True,
+        validators=[FileExtensionValidator(LOGO_ALLOWED_EXTENSION), validate_logo_size],
+    )
+    description = models.TextField()
+
+    def __str__(self):
+        return 'Business Line {}'.format(self.title)
+
+
 class UserApplication(TimeStampedModel):
     """
     Model for status of all required parts of user application submission.
     """
     user = models.OneToOneField(User, related_name='application', on_delete=models.CASCADE, )
     city = models.CharField(verbose_name=_('City'), max_length=255, )
+
+    saudi_national = models.BooleanField(default=True)
+    MALE = 'male'
+    FEMALE = 'female'
+    OTHER = 'other'
+
+    GENDER_CHOICES = (
+        (MALE, _('Male')),
+        (FEMALE, _('Female')),
+        (OTHER, _('Prefer not to answer')),
+    )
+    gender = models.CharField(choices=GENDER_CHOICES, max_length=8, default=OTHER,)
+    phone_number = PhoneNumberField(blank=True, region='SA')
+    birth_date = models.DateField(default=timezone.now)
+    business_line = models.ForeignKey(BusinessLine, on_delete=models.CASCADE, null=True)
```

Comment:
Shouldn't we use `OneToOneField`? ForeignKey means an Application can have many business lines.

If you make changes in models, please revert the new migration and recreate it